src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -425,15 +425,16 @@ def show_updates_available(new_line_before=False, new_line_after=False):
     if updates_available_components:
         if new_line_before:
             logger.warning("")
-        show_updates(updates_available_components)
+        show_updates(updates_available_components, only_show_when_updates_available=True)
```

Comment:
The method is called in places like `az xx --help`, only show version message when updates available.